core-services/prow/02_config/open-cluster-management-io/ocm-kustomize-generator-plugins/_prowconfig.yaml

```diff
@@ -13,7 +13,7 @@ branch-protection:
               required_status_checks:
                 contexts:
                 - Linting (1.16)
-                - Unit Tests (1.16)
+                - Unit Tests (1.18)
```

Comment:
We can probably skip this one for now. It needs a major update to keep up with midstream.

Reply:
Done, changed this back